static/app/views/explore/charts/index.tsx

```diff
@@ -129,26 +130,43 @@ export function ExploreCharts({query}: ExploreChartsProps) {
       {visualizes.map((visualize, index) => {
         const dedupedYAxes = dedupeArray(visualize.yAxes);
         const {chartType} = visualize;
+        const chartIcon =
+          chartType === ChartType.LINE
+            ? 'line'
+            : chartType === ChartType.AREA
+              ? 'area'
+              : 'bar';
```

Comment:
@Zylphrex Fixed it I don't even remember why I did it the other way... thnx for the patience